web/src/hashchange.js

```diff
@@ -323,7 +346,16 @@ function do_hashchange_overlay(old_hash) {
         if (base === "settings") {
             settings_panel_menu.normal_settings.activate_section_or_default(section);
         } else {
-            settings_panel_menu.org_settings.activate_section_or_default(section);
+            if (section === "user-list-admin") {
+                const right_side_tab = hash_parser.get_current_nth_hash_section(2);
+                const right_section = handle_invalid_users_section_url(right_side_tab);
+                settings_panel_menu.org_settings.activate_section_or_default(
+                    section,
+                    right_section,
```

Comment:
This can be better structured as -
```js
let right_section;
if (section === "user-list-admin") {
    const current_right_section = hash_parser.get_current_nth_hash_section(2);
    right_section = handle_invalid_users_section_url(right_side_tab);
}
settings_panel_menu.org_settings.activate_section_or_default(
    section,
    right_section,
);
```